packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Extends the `mime` package version constraint from `^1.0.4` to `>=1.0.4 <3.0.0`.
```

Comment:
I'm afraid that in order for users to pick this change up, it needs to be published to pub (which entails revving up the version in `pubspec.yaml` and updating the `CHANGELOG.md` of every modified package.)

Reply:
/me pretends he knows what he's doing in the packages repo.

Done. I think.

Reply:
The repo check was made `NEXT` was added to the CHANGELOG, but I don't _think_ packages/image_picker/image_picker needs any change, either the version or the CHANGELOG, since it doesn't have a dependency on mime.

Reply:
Giving it a shot -- federated packages still confuse the heck outta me. Presumably then we need to bump each of the underlying package versions/changelog individually?